src/redux/paymentDetailsSlice.js

```diff
@@ -0,0 +1,45 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { getPaymentDetails } from '../api';
+
+const initialState = {
+  paymentDetails: [],
+  paymentDetailsLoading: true,
+  paymentDetailsError: [],
+  activeOrBoundPaymentDetails: []
```

Comment:
as this is `paymentDetails` slice I would simplify the state to:
```
{
  paymentDetails: [],
  activeOrBoundPaymentDetails: [],
  loading: true,
  error: [],
}
```
I'm also thinking about 'activeOrBoundPaymentDetails' as these are payment methods that should be presented to customer (`paymentDetails` are all payment details returned from API) . If you have any idea for a better name for it - go ahead :)